Kconfig

```diff
@@ -119,6 +119,18 @@ config KAS_INCLUDE_EIO
 	default "kas/opt/eio.yml"
 	depends on EIO
 
+config META_HAILO
+	bool "Meta-Hailo AI card support"
+	help
+	  Enable building of meta-hailo, which includes the following main
+	  components: hailo-pci, hailo-firmware, hailort (hailocli, hailort,
+	  libhailort-dev, python3-hailort, libgsthailo). 
```

Comment:
Better to write this way: "include the hailo driver, firmware and userspace runtime"